src/utils/database-common/src/transactions/db_transaction_manager.rs

```diff
@@ -73,29 +71,30 @@ impl DatabaseTransactionRunner {
                 .add_value(transaction_ref.clone())
                 .build();
 
-            callback(catalog_with_transaction)
-                .instrument(tracing::info_span!("Transaction Body"))
-                .await
+            callback(catalog_with_transaction).await
```

Comment:
At least keep the event that transaction has started separately from the body statements.
I've seen that `sqlx` only has it's own events for `commit` phase, but not for `begin`.

Reply:
Transaction manager spams A LOT currently, so I wanted to trim it.

I see the value of knowing when transaction starts and ends, and whether it commits or rolls back, but I didn't understand the value of tracing the "body" separately.

What information do you think it gives us? Is it how long it took to open and commit transaction or something else?

My feeling was that such fine-grain tracing is crossing into "profiling" territory and maybe we should use metrics and histograms for fine timings instead.

Reply:
We might want to see that opening and closing transaction takes a lot of time. This might be an indication of heavy load on the connection pool, and it might need to be adjusted (more connections, tweaking timeouts, ..)

With Sqlite we might wait longer, i.e., as there is just 1 connection allowed in the pool.

Reply:
restored the span for now